### ISSUE TITLE
Escape key no longer ends drag after #401

### ISSUE THREAD

**Issue by billybednar** [dockpanelsuite/dockpanelsuite#457]
It's no longer possible to stop dragging dock content by pressing the escape key after #401.

**Comment by billybednar** [comment]
The check for WM_KEYDOWN does need to done as a message filter since the window begin dragged doesn't have keyboard focus or something like that (not too familiar with how input works on win32). The other three can probably stay in WndProc since the mouse is captured at the start of dragging, but I do wonder why they were originally written using a filter. 
```c#
protected virtual bool OnPreFilterMessage(ref Message m)
{
    if (m.Msg == (int)Win32.Msgs.WM_MOUSEMOVE)
        OnDragging();
    else if (m.Msg == (int)Win32.Msgs.WM_LBUTTONUP)
        EndDrag(false);
    else if (m.Msg == (int)Win32.Msgs.WM_CAPTURECHANGED)
        EndDrag(!Win32Helper.IsRunningOnMono);
    else if (m.Msg == (int)Win32.Msgs.WM_KEYDOWN && (int)m.WParam == (int)Keys.Escape)
        EndDrag(true);

    return false;
}
```

**Comment by lextm** [comment]
Before a full solution is found, I just disable the patch from #401 in master, 99b77161721a63f4527bceafa4870bba5aa1540c .

Will release 2.12 pretty soon to include this change.

**Comment by lextm** [comment]
Closed temporarily and wait for a better patch to replace #401.
